opm/common/utility/numeric/calc_cellvol.hpp

```diff
@@ -0,0 +1,7 @@
+#include <vector>
```

Comment:
This file (and the cpp file) still missing copyright/license header. The one for the test program is fine though!